.github/workflows/preview.yml

```
@@ -0,0 +1,65 @@
+name: Expo Preview
+on:
+  push:
+    branches: [master, main]
+jobs:
+  changed:
+    name: Find changed apps
+    runs-on: ubuntu-latest
+    outputs:
+      exampleChanged: ${{ steps.filter.outputs.examples == 'true' }}
+      exampleNames: ${{ steps.find.outputs.result }}
+    steps:
+      - id: filter
+        name: 🕵️ Detect changes
+        uses: dorny/paths-filter@v2
+        with:
+          list-files: json
+          filters: |
+            examples:
+              - blank
+              - navigation
+              - with-**
+      - id: find
+        name: 📱 Find changed examples
+        uses: actions/github-script@v5
+        if: steps.filter.outputs.examples == 'true'
+        with:
+          script: |
+            // Load all changed files to dynamically fetch the app names
+            const path = require('path')
+            const changes = ${{ steps.filter.outputs.examples_files }}
+            // Iterate through all changes and keep unique app names
+            return [...new Set(changes.map(file => path.dirname(file).split('/', 1)[0]))]
```

```suggestion
            return [...new Set(changes.map(file => path.dirname(file).split('/', 1)[0]))].map(
                name => name === 'with-yarn-workspaces' ? 'with-yarn-workspaces/apps/mobile' : name
            )
```

It's wack but does solve the "different structure" issue. Alternatively, we could run a yarn script, but that will include the actual script in the example files people download.

---

Resolved by ignoring in the filter